web/templates/policy/coc.html.md

```diff
@@ -0,0 +1,93 @@
+## Code of Conduct
+
+Hex exists to facilitate sharing code, by making it easy for Erlang ecosystem developers to publish and distribute packages.
+
+Hex is a piece of technology, but more importantly, it is a community.
+
+We believe that our mission is best served in an environment that is friendly, safe, and accepting; free from intimidation or harassment.
+
+Towards this end, certain behaviors and practices will not be tolerated.
+
+Hex is the set of formats and APIs that makes up the specification of Hex. Hex.pm is an implementation of the Hex specifications, it is comprised of this website and associated HTTP API, the package repository including packages published to Hex.pm. In this document Hex and Hex.pm is used interchangeably but it refers to Hex.pm.
+
+### tl;dr
+
+  * Be respectful.
+  * We're here to help: <support@hex.pm>
+  * Abusive behavior is never tolerated.
+  * Data published to Hex is hosted at the discretion of the Hex team, and may be removed.
+  * Violations of this code may result in swift and permanent expulsion from the Hex community.
+
+### Scope
+
+We expect all members of the Hex community, including paid and unpaid agents, administrators, and users to abide by this Code of Conduct at all times in all Hex community venues, online and in person, and in one-on-one communications pertaining to Hex affairs.
+
+This policy covers the usage of the Hex registry, as well as the Hex website, Hex related events, and any other services offered by or on behalf of Hex (collectively, the "Service"). It also applies to behavior in the context of the Hex Open Source project communities, including but not limited to public GitHub repositories, IRC channels, social media, mailing lists, and public events.
```

Comment:
As a concerned observer of the [Opal debacle](https://github.com/opal/opal/issues/941), I would like to know whether the Hex team intends to interpret these clauses as applying to anyone who contributes a hex package in their personal communication, such as on their personal Twitter account or blog.

I love Elixir and Hex, and I would hate to see valuable contributions rejected or pulled from Hex because the contributor held views that were not PC.